### PR TITLE
WIP Delete booking

### PR DESCRIPTION
**Pull request by lollypop27**
Delete an existing booking

Link to PT story: 
https://www.pivotaltracker.com/story/show/130533397

Description of the PR:

1. Create a route
2. Create a action in dashboard controller
3. Add a delete button when modal is in edit mode
4. Add translation in en and sv YML files

Ready for review!
@tochman @diraulo

### CODE REVIEW

File: app/views/dashboard/_dashboard.js.erb [hunk 1]
```diff
@@ -148,7 +148,9 @@ function populateModal(object) {
     $('#booking_time_end').val(booking_times[1]);
     $('#booking-form').attr('action', '<%= edit_booking_path %>').attr('method', 'post');
     $('#booking-form').append('<input type="hidden" name="booking[original_time_start]" value="' + booking_times[0] + '" />');
-    $('#booking-form').find(':submit').val('Update');
+    $('#booking-form').find('#update').val('Update');
```
Comment (tochman) [inline app/views/dashboard/_dashboard.js.erb]: Can we add a translation for this?

File: app/controllers/dashboard_controller.rb [hunk 2]
```diff
@@ -46,6 +46,11 @@ def edit_booking
     redirect_to root_path(date: booking_params[:booking_date])
   end
 
+  def delete_booking
+    @resource.destroy
```
Comment (tochman) [inline app/controllers/dashboard_controller.rb]: We don't want to destroy the resource, jut the booking.

Comment (lollypop27) [inline app/controllers/dashboard_controller.rb]: @tochman ok, also not sure what happened but the delete button is now showing when you first create a booking and wasn't showing before it only showed when one clicked to edit booking?

Comment (tochman) [inline app/controllers/dashboard_controller.rb]: skarmavbild 2016-09-25 kl 18 58 44
skarmavbild 2016-09-25 kl 18 58 54

Comment (tochman) [inline app/controllers/dashboard_controller.rb]: Not on my version of the code. My latest changes from yesterday fixes all issues and should not cause you any errors. Do a forced pull from your origin `git pull origin delete_booking -f` That should give you the latest changes without any merging conflicts. Make sure you run a local server and create a booking. After that you should be able to delete it.

**Comment by tochman** [comment]
Imo this is ready for merge

**Pull request merged**